parsers/TR.py

```diff
@@ -33,7 +41,14 @@
     "wind": ["wind"],
     "coal": ["blackCoal", "asphaltiteCoal", "lignite", "importCoal"],
     "hydro": ["river", "dammedHydro"],
+    "nuclear": ["nucklear"],
```

Comment:
And now we will start collecting nuclear data as soon as the plats are completed (and they start reporting it)!